### DISCUSSION TITLE
Wp-Adv-Quiz v1.0.2

### DISCUSSION THREAD

**Discussion by markusbegerow** [markusbegerow/Wp-Adv-Quiz#40] [Announcements]
### Wp-Adv-Quiz v1.0.2
I'm pleased to release a new version of Wp-Adv-Quiz.
### What's new?
- WordPress 6.3 and PHP 8.2 support and tested
- New minor bugfixes
- Gutenberg block fixed
- New export statistics features available (CSV, XLS, JSON)
### Who should upgrade?
Everyone who wants to run the quiz with Wp-Adv-Quiz v1.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/markusbegerow/Wp-Adv-Quiz/releases/tag/v1.0.2'>Wp-Adv-Quiz v1.0.2</a>.</em>

**Comment by brandonc89** [comment]
Hi Markus,

This is great, thank you for all the work! I will definitely be donating to the cause. 
But here is my question, I had the old wp-pro-quiz with a backup of all my exams in XML format. My website experienced a crash and I lost wp-pro-quiz in the process. The XML file is 9Mb and Wp-adv-quiz will not let me upload a file of that size.
Is there any other way to get all of my content into wp-adv-quiz? It would take me weeks to recreate everything so anything would be helpful.
Can I upload my old wp-pro-quiz folder into /wp-content/plugins or something too because I have a backup of that from prior to the crash?

Thank you!

**Comment by markusbegerow** [comment]
@brandonc89 please check your issue page. I've provided some solutions for your case
